third_party/proton/csrc/lib/Data/TraceData.cpp

```diff
@@ -448,7 +448,7 @@ void dumpKernelMetricTrace(
 } // namespace
 
 void TraceData::dumpChromeTrace(std::ostream &os) const {
-  auto events = trace->getEvents();
+  auto& events = trace->getEvents();
```

Comment:
`getEvents` explicitly returns a `std::vector<TraceEvent> &`, but `auto events` deduces the `events` as a value, leading to an extra copy.